calculators/gas_king_rating.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+$GLOBALS['rating_calculators']['_mignoubou_gas_king'] = [
+    'description' => '
+        by <a href="/player/22267">PewPewPew</a>: gas king rating v1
+        only objective: gas as much and as soon as you can
+
+        you start with 2000 rating (compression is activated although maybe not efficient, I was too lazy to see how you did it)
```

Comment:
this text is a description for the public page like https://ds-rating.com/rating/level 
![image](https://github.com/dima-stefantsov/dsr_ratings/assets/1370909/1d6c3495-989f-41ab-9bdb-95c615d3361a)

you probably don't want to announce how lazy you are, and definitely people reading will not understand what they did that you mention.

Reply:
OH ok I didn't know, I'll definitely change this then yes.

Reply:
People won't be blatantly aware that I'm lazy anymore, fixed